insights/client/apps/ansible/__init__.py

```diff
@@ -10,13 +29,31 @@ def verify_playbook(unverified_playbook: str) -> str:
     Output: stringified "verified" playbook
     Error: exception
     """
-    # Skeleton implementation ... "bless" the incoming playbook
+    # Skeleton implementation ... "bless" the incoming playbook 
+    ERROR = os.getenv('THROW_ERROR')
```

Comment:
What do you think about making this a bit more verbose?  Something like "ANSIBLE_PLAYBOOK_VERIFIER_THROW_ERROR"?  Its probably overly verbose.  I tend to lean towards being overly verbose in code.

Reply:
That sounds great to me!  I'll change it to something like that!